models/datamapper_setup.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra'
+require 'data_mapper'
+require 'dm-postgres-adapter'
+require "dm-validations"
+
+configure :development do
+  DataMapper.setup(:default, "postgres://localhost/chitter_dev")
+end
+
+configure :test do
+  DataMapper.setup(:default, "postgres://localhost/chitter_test")
```

Comment:
It would be even cleaner to get the database URLs from environment variables.  eg `ENV["production_database_url"]`.  This would make deploying easier in the future, and allow you to keep the URLs secret (having them public is a mild security leak).